draft-ietf-rtcweb-jsep.xml

```diff
@@ -1699,7 +1702,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             DTLS certificate fingerprints.</t>
 
             <t>
-            <xref target="RFC4568"></xref> MUST NOT be implemented to
+            <xref target="RFC4568"></xref> SHALL NOT be implemented to
```

Comment:
Would it make sense to move this to the end? Seems a bit odd to encounter it in the middle.

Also, we might want to adjust the prose here to say something like

> The SDES SRTP keying mechanism from RFC 4568 SHALL NOT be implemented, as discussed in <xref security-arch/>.

This might also obviate the need for SHALL NOT, which feels clunky to me.